### ISSUE TITLE
Campaign action triggered Text Message  ( Last execution error: Scheduled )

### ISSUE THREAD

**Issue by josh-hernd** [mautic/mautic#10712]
### Mautic Version

4.0.0

### PHP version

7.4.26

### What browsers are you seeing the problem on?

Not relevant

### What happened?

# Abstract Issue
Mautic successfully sends text messages triggered by a Campaign Action **( only once )**.

On a **( different / New )** Campaign with action to trigger a  **( New )** text message, it encounters 
an error **( Last execution error: Scheduled )** found on **( contact view details )**.

Campaigns are working accordingly except for the ( text messages after the first one sent ).

# Mautic Version 
4.1.0 selection not available, forced to select 4.0.0. 

# Environment
* Google Cloud Platform VM instance 
* Image bitnami-mautic-4-1-0-7-r01-linux-debian-10-x86-64-nami

## Twilio Enabled/Auth
* Enabled and published
* SID and Auth is correct
## Twilio Features
* Sending Phone Number: has correct number
* Do not send text message more than:  is blank 
* messages each: set to nothing

# Error Log

```
[2021-12-28 18:44:13] mautic.NOTICE: PHP Notice - Undefined index: path - in file /opt/bitnami/mautic/app/bundles/CoreBundle/Controller/ExceptionController.php - at line 120 {"request":"[object] (Symfony\\Component\\HttpFoundation\\Request: GET //console HTTP/1.1\r\nAccept-Encoding:       gzip\r\nConnection:            Keep-Alive\r\nHost:                  mywebsite.com\r\nReferer:               https://localhost:443/console/\r\nUser-Agent:            Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.108 Safari/537.36\r\nVia:                   1.1 google\r\nX-Cloud-Trace-Context: b9edcd1bda971d59ee5c19987d39c4d4/16518456462588559379\r\nX-Forwarded-For:       195.54.160.149, localhost\r\nX-Forwarded-Proto:     https\r\nX-Php-Ob-Level:        1\r\n\r\n)","exception":"[object] (Symfony\\Component\\Debug\\Exception\\FlattenException: {})","logger":null,"class":"Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException","currentContent":"","layout":"Error","code":404,"anonymous":true,"baseTemplate":"MauticCoreBundle:Default:slim.html.php","templatePage":false,"template":"MauticCoreBundle:Error:404.html.php","templating":"[object] (Symfony\\Bundle\\FrameworkBundle\\Templating\\DelegatingEngine: {})","statusText":"Not Found","url":"//console","urlParts":{"host":"console"}} []
[2021-12-28 18:44:20] mautic.NOTICE: PHP Notice - Undefined index: path - in file /opt/bitnami/mautic/app/bundles/CoreBundle/Controller/ExceptionController.php - at line 120 {"request":"[object] (Symfony\\Component\\HttpFoundation\\Request: GET //console HTTP/1.1\r\nAccept-Encoding:       gzip\r\nConnection:            Keep-Alive\r\nHost:                  mywebsite.com\r\nReferer:               https://localhost:443/console/\r\nUser-Agent:            Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.108 Safari/537.36\r\nVia:                   1.1 google\r\nX-Cloud-Trace-Context: d934ad15ec3214d1d40b9f8ea764f45a/2580157875413020646\r\nX-Forwarded-For:       195.54.160.149, localhost\r\nX-Forwarded-Proto:     https\r\nX-Php-Ob-Level:        1\r\n\r\n)","exception":"[object] (Symfony\\Component\\Debug\\Exception\\FlattenException: {})","logger":null,"class":"Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException","currentContent":"","layout":"Error","code":404,"anonymous":true,"baseTemplate":"MauticCoreBundle:Default:slim.html.php","templatePage":false,"template":"MauticCoreBundle:Error:404.html.php","templating":"[object] (Symfony\\Bundle\\FrameworkBundle\\Templating\\DelegatingEngine: {})","statusText":"Not Found","url":"//console","urlParts":{"host":"console"}} []
[2021-12-28 15:03:49] mautic.NOTICE: PHP Notice - Trying to access array offset on value of type null - in file /opt/bitnami/mautic/app/bundles/SmsBundle/Form/Type/SmsSendType.php - at line 84 {"builder":"[object] (Symfony\\Component\\Form\\FormBuilder: {})","options":{"block_name":null,"disabled":false,"label_format":null,"label_translation_parameters":[],"attr_translation_parameters":[],"translation_domain":null,"auto_initialize":true,"trim":true,"required":true,"property_path":null,"mapped":true,"by_reference":true,"inherit_data":false,"compound":true,"method":"POST","action":"","post_max_size_message":"The uploaded file was too large. Please try to upload a smaller file.","allow_file_upload":false,"help_translation_parameters":[],"error_mapping":[],"invalid_message":"This value is not valid.","invalid_message_parameters":[],"allow_extra_fields":false,"extra_fields_message":"This form should not contain extra fields.","csrf_protection":true,"csrf_field_name":"_token","csrf_message":"The CSRF token is invalid. Please try to resubmit the form.","csrf_token_manager":"[object] (Symfony\\Component\\Security\\Csrf\\CsrfTokenManager: {})","csrf_token_id":null,"block_prefix":null,"label":false,"row_attr":[],"attr":[],"data":null,"data_class":null,"empty_data":"[object] (Closure: {})","error_bubbling":true,"label_attr":[],"upload_max_size_message":"[object] (Closure: {})","help":null,"help_attr":[],"help_html":false,"validation_groups":null,"constraints":[],"update_select":"campaignevent_properties_sms"},"windowUrl":"/s/sms/new?contentOnly=1&updateSelect=campaignevent_properties_sms"} []
```

# Comment 
There is a possibility i am doing something incorrect, but if is a true bug please advice me and others on how to fix such bug.

### How can we reproduce this issue?

Step 1: Add a contact with phone number
Step 2: Create Segment and add contacts
Step 2: Create campaign ( In Campaign Builder ) Select Segment => Action ( Send Text message )

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

**Comment by dralawady** [comment]
I'm having the same problem. it sounds that system-wide frequncy rules for Twilio are not applied even after changing them.

**Comment by RCheesley** [comment]
This was addressed by https://github.com/mautic/mautic/pull/10753 which will be released in 4.2 at the end of February - sorry for the late triage of the issue!
